src/lib/pmd/PmdEngine.ts

```diff
@@ -88,7 +89,7 @@ abstract class BasePmdEngine implements RuleEngine {
 			return results;
 		} catch (e) {
 			this.logger.trace('Pmd evaluation failed: ' + (e.message || e));
-			throw new SfdxError(e.message || e);
+			throw new SfdxError(this.processStdErr(e.message || e));
```

Comment:
#processStdError expects a string, I believe the `|| e` clause will cause a runtime error because `Exception.includes` will be undefined.